app/js/views/edit.js

```diff
@@ -152,6 +157,21 @@ export default class EditView extends View {
     });
 
     this.htmlTextarea.value = html;
+    this.scriptTextarea.value =
+`/**
+ * You can directly edit the generated
+ * add-on script from here.
```

Comment:
This comment is inaccurate as we're supplementing the scripts generated through other operations, not editing/replacing them.